content/github-staff/github-shared-components.mdx

```diff
@@ -21,3 +24,5 @@ Please note that these components are **owned by the feature teams**, and will n
 
 - [Handling new patterns](/guides/contribute/handling-new-patterns)
 - [Upstreaming to Primer](/guides/contribute/adding-new-components#upstreaming-to-primer)
+
+## Shared components
```

Comment:
The links in this section don't match Primer's link styles.

<img width="416" alt="Screenshot 2023-08-15 at 5 40 55 PM" src="https://github.com/primer/design/assets/2313998/f519f3b1-c33c-4d83-9c82-6522b26fc7e9">